lib/controllers/cats.js

```diff
@@ -1,3 +1,26 @@
 const { Router } = require('express');
+const { cats } = require('../cats-data');
 
-module.exports = Router();
+module.exports = Router()
+  .get('/welcome', (req, res) => {
+    console.log('in welcome greeting');
+    res.json(cats);
+  })
+  .get('/:id', (req, res) => {
+    let match = '';
+    for (const kitty of cats) {
+      if (kitty.id === req.params.id) {
+        match = kitty;
+      }
+    }
```

Comment:
For loops are for chumps. I'd prefer you had used one of the array methods you know and love. In this case a `.find()` would suit you well. 

```js
const match = cats.find(cat => cat.id === req.params.id);